admin/class-admin-apple-themes.php

```diff
@@ -159,6 +159,28 @@ public static function render_field( $theme, $option_name ) {
 		);
 	}
 
+	/**
+	 * Return a Theme object loaded with db values based on the theme name string.
+	 *
+	 * @param string $theme_str The name of theme.
+	 * 
+	 * @return \Apple_Exporter\Theme|null $theme_object A Theme object loaded with db config,
+	 * or null if theme name not in registry.
+	 */
+	public static function get_loaded_theme( $theme_str ) {
+		// Make sure theme exists in registry before continuing.
```

Comment:
The `load` function should be robust enough to handle a case where a theme is requested that doesn't exist, so we can save some processing time by not checking the registry first.